### PR TITLE
Reordered scheme initialization

### PR DESCRIPTION
**Pull request by arjunrn**
The scheme initialization was done after configuring the metrics which was causing and error. Just moved the the relevant lines to before the metrics part.

Also registered metrics with prometheus metrics registry correctly.

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -129,6 +143,7 @@ func main() {
 		WithPath("/metrics").
 		WithPort(strconv.Itoa(metricsPort)).
 		WithServiceMonitor().
+		WithCollectors(metrics.GetMetricsAggregator().GetMetrics()).
```
Comment (2uasimojo) [inline cmd/manager/main.go:146]: This part looks unrelated to the topic, and nontrivial.

Comment (arjunrn) [reply cmd/manager/main.go:146]: I can move it into another PR but basically it was an oversight from the first refactoring I did in this repo. I can move it into another PR.

**Review by 2uasimojo — APPROVED** [review]
Thanks.

/lgtm
/approve
